backend/database/connection.go

```diff
@@ -0,0 +1,61 @@
+package database
+
+import (
+	"backend/config"
+	"backend/models"
+	"fmt"
+
+	"golang.org/x/crypto/bcrypt"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
+
+func Connect() {
+	connection, err := gorm.Open(postgres.New(postgres.Config{
+		DSN: config.GetPostgresConnectionStr(),
+	}), &gorm.Config{})
+
+	if err != nil {
+		panic("could not connect to the database")
+	}
+	DB = connection
+	ClearTables(connection)
```

Comment:
Is this deleting all the users on every connect?

Reply:
yup I was using this code for testing, gonna remove it now